utils/retrieve-state/src/main.rs

```diff
@@ -79,14 +83,26 @@ struct Opts {
     use_gzip: bool,
 
     #[structopt(
-        short,
+        short = "b",
         long,
         about = "Specify the path to the folder to be used for loading block files to be converted."
     )]
     block_file_download_path: Option<PathBuf>,
 
-    #[structopt(short, long, about = "Enable manual syncing after each block to LMDB")]
-    manual_sync_enabled: bool,
+    #[structopt(long, about = "Enable manual syncing after each block to LMDB")]
+    manual_sync_enabled: Option<bool>,
+
```

Comment:
It might be a good idea to have an option for how many workers. But we'd have to drop the `tokio::main` macro and use the builder in that case.